### ISSUE TITLE
Buttons do not indicate focus for keyboard access

### ISSUE THREAD

**Issue by LaCuneta** [NetLogo/NetLogo#2531]
## Steps to Reproduce

- Open the Models Library, the search input box should be highlighted/blinking cursor
- Hit tab to move focus to the next control, observe the Clear button is not highlighted
- Continuing hitting tab; the tree view and text input seem to give an indication they are focused, the buttons do not.
- Note that the buttons are gaining focus, as hitting enter after tabbing will cause them to activate, it's just not visually apparent.

## What Happens in 6.4

In NetLogo 6.4 focused button elements do gain a highlight indicator/border.

## Notes

While the steps to reproduce reference the Models Library, it appears to affect all buttons, as on the model save/discard dialog, shape editors, etc.  Tested on Ubuntu 24, 64-bit, light or dark mode.

**Comment by ERSUCC** [comment]
We had some difficulties finding simple button focus indicators that worked well with the themes we originally designed, so we decided to tackle it after 7.0 to save time in design meetings. I agree that buttons should have focus indicators.
